### PR TITLE
Plot fluid surrogate solution

### PR DESCRIPTION
**Pull request by aprilnovak**
Added capability to plot the fluid surrogate T/H solution, simultaneously with the solid solution or independently.

I haven't tested this yet for multi-pin problems because there wasn't an example ready-to-go in the repo. If anyone has a smallish multi-pin problem that they can check in, that'd be great, otherwise I'll set one up to verify the plotting works.

### CODE REVIEW

**Comment by pshriwise** [comment]
Sorry for the delay, hoping to look this over by the end of the week!

**Comment by paulromano** [comment]
Sorry for the delayed on this @aprilnovak -- I think I have a multi pin case ready to go from our last milestone report so I'll give it a whirl on that!

File: src/vtk_viz.cpp [hunk 1]
```diff
@@ -408,37 +508,92 @@ xtensor<double, 3> SurrogateVtkWriter::clad_points()
...
+  // create ring on clad surface to set points on clad surface
+  xtensor<double, 2> ring = create_ring(surrogate_.clad_outer_radius(), azimuthal_res_);
+
+  xt::view(x, xt::range(0, azimuthal_res_)) = xt::view(ring, 0, xt::all());
+  xt::view(y, xt::range(0, azimuthal_res_)) = xt::view(ring, 1, xt::all());
+
+  // set remaining points on boundary
+  double half_pitch = surrogate_.pin_pitch() / 2.0;
+  for (size_t i = 0; i < 8; ++i) {
+    double index = i + azimuthal_res_;
```
Comment (paulromano) [inline src/vtk_viz.cpp]:
```suggestion
    size_t index = i + azimuthal_res_;
```

**Comment by aprilnovak** [comment]
Thanks @paulromano for the commit suggestion :) All the power outages on the Berkeley campus mean that the server I do all this stuff on is out of commission so you saved me a few steps!

**Comment by pshriwise** [comment]
Testing it out now!

**Comment by pshriwise** [comment]
Works great! Thanks @aprilnovak!

**Pull request merged**